docs/tiller_ssl.md

```diff
@@ -312,6 +312,6 @@ If your tiller certificate has expired, you'll need to sign a new certificate, b
 
 ## References
 
-https://github.com/denji/golang-tls
-https://www.openssl.org/docs/
+https://github.com/denji/golang-tls  
```

Comment:
Thanks @pfuhrmann for the cleanup. I think it might be worth making the references a list :
- https://github.com/denji/golang-tls 
.....
What do you think?

Reply:
@hickeyma good idea, changed to the list, also changed PR's title to keep it consistent